testsuite/features/secondary/proxy_retail_pxeboot_and_mass_import.feature

```diff
@@ -215,6 +215,14 @@ Feature: PXE boot a Retail terminal
     Then I should see a "1 package install has been scheduled" text
     When I wait until event "Package Install/Upgrade scheduled by admin" is completed
 
+  Scenario: Chect that the PXE boot minion stays configured after reboot
```

Comment:
`Chect that the PXE boot minion stays configured after reboot`
 =>
`The PXE boot minion stays configured after reboot`